core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/WorkspaceManagerDAO.scala

```diff
@@ -19,6 +19,12 @@ trait WorkspaceManagerDAO {
                                       ctx: RawlsRequestContext
   ): CreatedWorkspace
 
+  def createProtectedWorkspaceWithSpendProfile(workspaceId: UUID,
```

Comment:
Nit: instead of a separate method, I would have just added an argument to `createWorkpaceWithSpendProfile`.